packages/kotti-ui/source/kotti-field-select/constants.ts

```diff
@@ -31,6 +31,18 @@ export const KOTTI_FIELD_SELECT_SUPPORTS: KottiField.Supports = {
 
 export const KOTTI_FIELD_SINGLE_SELECT_PROPS = {
 	...KOTTI_FIELD_SELECT_PROPS,
+	actions: {
+		default: () => null,
```

Comment:
would never recommend a default `null` for an array

Reply:
Additionally, you don’t need to use a function for primitive defaults (like `null`)

Yeah, this should be `() => []`